UPGRADING.rst

```diff
@@ -95,3 +95,20 @@ Overview of removed Graylog web interface settings:
 +---------------------------------+----------------------------------+--------------------+
 | ``web_tls_key_password``        | ``http_tls_key_password``        | Empty              |
 +---------------------------------+----------------------------------+--------------------+
+
+Plugins moved into the Graylog server
+=====================================
+
+In 3.0 we merged the following plugins into the server Graylog server repository:
```

Comment:
Duplicate "server".

Alternative wording:
"Starting with Graylog 3.0.0, the following official plugins were merged into Graylog core:"